src/renderer/delta-remote.ts

```diff
@@ -259,6 +259,18 @@ class DeltaRemote {
   call(fnName: string, ...args: any[]): Promise<any> {
     return _callDcMethodAsync(fnName, ...[...arguments].slice(1))
   }
+
+  on(channel: string, listener: (event: Electron.IpcRendererEvent, ...args: any[]) => void) {
+    ipcBackend.on(channel, listener)
```

Comment:
```suggestion
  on(...args: Parameters<typeof ipcRenderer.once>) {
    ipcBackend.on(...args)
```
you can simplify this.